Microsoft.NET.Build.Containers/ParseContainerProperties.cs

```diff
@@ -74,6 +74,20 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
+        if (!ContainerRegistry.StartsWith("http://") &&
+             !ContainerRegistry.StartsWith("https://") &&
+             !ContainerRegistry.StartsWith("docker://"))
+        {
+            // Default to https when no scheme is present: https://github.com/distribution/distribution/blob/main/reference/normalize.go#L88
```

Comment:
Nit: you can get a permalink to a specific commit by hitting `y` on any source page on GitHub, and those are better in case they move stuff around/refactor.

```suggestion
            // Default to https when no scheme is present: https://github.com/distribution/distribution/blob/26163d82560f4dda94bd7b87d587f94644c5af79/reference/normalize.go#L88
```